airbyte-server/src/main/resources/application.yml

```diff
@@ -2,23 +2,13 @@ micronaut:
   application:
     name: airbyte-server
   security:
-    intercept-url-map:
-      - pattern: /**
-        httpMethod: GET
-        access:
-          - isAnonymous()
-      - pattern: /**
-        httpMethod: POST
-        access:
-          - isAnonymous()
-      - pattern: /**
-        httpMethod: HEAD
-        access:
-          - isAnonymous()
+    authentication-provider-strategy: ALL
+    enabled: ${API_AUTHORIZATION_ENABLED:false}
```

Comment:
If I understand well, this will disable the authentication on OSS

Reply:
Correct.  By default, it will disable Micronaut's built-in security support.  If we set it to true in OSS and/or in Cloud, it will enable it.  The plan will be to set the env var in the Helm charts for the airbyte-server in Cloud to `true`.